### DISCUSSION TITLE
Hardware incompatability with Ver 0.22.2?

### DISCUSSION THREAD

**Discussion by bquock** [Stellarium/stellarium#2653] [General]
I love Stellarium and use it on several different computers, but I just installed it on a Win 11 Lenovo, and it continually blinks out for a few seconds, then comes back on.  A couple of times, it has crashed entirely while I was trying to configure extras & plugins to my liking.  No other programs on the same computer do this, and Stellarium doesn't behave this way on other computers that I have it installed on.  Any suggestions or advice would be much appreciated.  Not a programmer, so please go easy on the lingo.  Thanks!

**Comment by alex-w** [comment]
Please check the drivers for graphics card - probably you need to install it from website of manufacturer

**Reply by bquock** [reply]
Thanks for the suggestion, but the computer said I already had the latest driver.  However, someone else suggested running the program in compatibility mode, and I had the option to run it emulating Windows 8, and so far, I haven't seen the same kind of annoying blink-out that I was seeing previously.  Hopefully, that did the trick!  I wonder if anyone else has had issues running the program in Windows 11?

**Reply by gzotti** [reply]
We have occasional reports of screen blinking. However, these reports are usually technically incomplete, and we cannot reproduce the issues on our systems. One contributed patch that seems to solve the issue will be available in the next release.

**Comment by bquock** [comment]
That’s great to hear.  I wish I were technically proficient enough to offer more helpful info, but thanks for the response, and I look forward to the next release!

**Reply by alex-w** [reply]
Could you check the fresh beta - https://github.com/Stellarium/stellarium-data/releases/tag/weekly-snapshot ?
